manager/manager.go

```diff
@@ -110,10 +112,10 @@ type Manager interface {
 	CloseEventChannel(watch_id int)
 
 	// Get status information about docker.
-	DockerInfo() (DockerStatus, error)
+	DockerInfo() (docker.DockerStatus, error)
```

Comment:
Given that kubelet consumes these methods, should the API types be kept in a separate package?